dockerfiles/run/guice/cassandra-ldap/destination/conf/elasticsearch.properties

```diff
@@ -25,8 +25,22 @@
 elasticsearch.masterHost=elasticsearch
 elasticsearch.port=9200
 
-# Optional. Only http or https are accepted, default is http
-# elasticsearch.hostScheme=http
+# Optional, default is `default`
+# Choosing the SSL check strategy when using https scheme
+# default: Use the default SSL TrustStore of the system.
+# ignore: Ignore SSL Validation check (not recommended).
+# override: Override the SSL Context to use a custome TrustStore containing ES server's certificate.
+# elasticsearch.hostScheme.https.sslValidationStrategy=default
+
+# Optional. Required when using 'https' scheme and 'override' sslValidationStrategy
```

Comment:
why did you remove the hostScheme options then? (https://github.com/linagora/james-project/pull/2855/commits/dc05b722e2adc7f628ff406edd3bc1d1b6985bab#diff-ab4c2c6737d5ca0469265cb4b172a634L28 )

Reply:
sorry, my copy paste mistake